src/components/sidebar/sidebar.scss

```diff
@@ -2,12 +2,12 @@
 
 .pipeline-sidebar {
   position: absolute;
-  top: 0;
+  top: -1px; /* Avoids pixel rounding gaps */
+  bottom: -1px;
```

Comment:
😮

Reply:
Yeah while working on the minimap I'd noticed there's a missing pixel gap around the toolbar, so you could see a bit of the diagram under it.

I think it's usually due to transforms and GPU rounding type issues, so this is a good idea to take care of it as it can be a little distracting, if that makes sense?

Reply:
Yup! Interesting. Thanks for fixing 👍